core/src/test/java/com/google/cloud/sql/core/CloudSqlInstanceConcurrencyTest.java

```diff
@@ -214,7 +213,7 @@ private Thread startForceRefreshThread(CloudSqlInstance inst) {
     return t;
   }
 
-  private RateLimiter<Object> newRateLimiter() {
-    return RateLimiter.burstyBuilder(2, Duration.ofMillis(50)).build();
+  private RateLimiter newRateLimiter() {
+    return RateLimiter.create(20.0); // 20/sec = every 50 ms
```

Comment:
Shouldn't this match our production code?

Reply:
No. The concurrency tests reduce length of both modeled response times and timeouts from real behavior so that the tests run faster and we get more "shots on goal" to produce a deadlock or other threading problem.